AlembicImporter/Packages/com.unity.formats.alembic/Documentation/com.unity.formats.alembic.md

```diff
@@ -6,7 +6,7 @@ The Alembic package supports the import and playback of Meshes, Points and Camer
 
 ## Requirements
 
-The Alembic Package is compatible with Unity Editor 2018.2 and above.
+The Alembic Package is compatible with Unity Editor 2018.1 and above.
 
 The package is available on 64-bit desktop platforms:
 * Windows 10
```

Comment:
(http://support.ricoh.com/bb/common/Trademark.html)
Microsoft® Windows® 10

Reply:
(https://www.apple.com/legal/intellectual-property/trademark/appletmlist.html)
macOS® Sierra (10.12)

(https://www.centos.org/legal/trademarks/)
(https://www.ubuntu.com/legal/terms-and-policies/intellectual-property-policy)
GNU/Linux (CentOS 7, Ubuntu 16.x and Ubuntu 17.x)

Reply:
(can't add comments on unmodified lines..)
Line 18: remove trailing dot

Reply:
line 30: ... the Time property on the Alembic Stream Player component.

Reply:
line 34: Reword it a bit
"
To animate using the Timeline first create a Timeline component on a Game Object (q: does it need to be the Alembic Game Object?). You can then animate the Alembic Stram Player Time property using an infinite clip as shown below. 
"

Reply:
the Timeline component is called PlayableDirector. Maybe if I just say create a timeline asset? do we need to say how to create it?

Reply:
This is well explained in a section below (Record and playback the Alembic using an Infinite Clip):
"
You can control the playback of Alembic using an Infinite Clip on a Timeline Animation Track bound to the game object with the Alembic Stream Player component
"

Reply:
I would not bother telling how to create a timeline asset.